### PR TITLE
Resolving merge conflicts with simrin_joshi.md file

### PR DESCRIPTION
**Pull request by simrin051**
I am a passionate professional focused on full stack development. I have 1.5 years of experience in Angular, HTML, CSS , JS,  Java and MySQL.  I am looking forward to actively contribute to Twindle.

## Attach Screenshot

> Note 2 code reviewer approval needed. Approach in twitter group & discord channel.

### CODE REVIEW

File: docs/TEAM.md [hunk 1]
```diff
@@ -89,3 +91,6 @@
 |                                                          | [Pratham](https://github.com/prasoonpratham)  | 2+ experience in JavaScript, always open to help :smile:                                           |
 |                                                          | [Sarath](http://http://www.sarathnagaraj.ca/) | 6+ experience in JavaScript, Currently working on a side project using ReactJS :man_technologist:  |
 |![Puru's profile pic](https://github.com/puruvj.png?size=80) | [Puru Vijay](https://github.com/puruvj) | 5+ experience in HTML, CSS, JS, Angular, React, Svelte, NodeJS, Python, StencilJS, Firebase, feel free to ping me for help 😁 | 
+
+
+-------------------------------------------------------------------------------------------------- |
```
Comment (PuruVJ) [inline docs/TEAM.md:96]: Please remove this long line, it's showing up in rich preview of this markdown file

**Review by Akshay2996 — APPROVED** [review]

**Comment by PuruVJ** [comment]
All right these can be merged. Those extra lines can be fixed anytime, no big issue.

**Review by joelvinaykumar — APPROVED** [review]

**Review by Akshay2996 — APPROVED** [review]
@proful Merge this PR too 🙂

**Review by joelvinaykumar — APPROVED** [review]
Alright

**Pull request merged**